.github/workflows/linters.yml

```diff
@@ -0,0 +1,67 @@
+name: Linters
+on: [push, pull_request]
+permissions: {}
+jobs:
+  check:
+      runs-on: ubuntu-22.04
+      steps:
+        - uses: actions/checkout@v3
+        - uses: actions-rs/cargo@v1
+          with:
+            command: check
+            args: --target "x86_64-unknown-linux-gnu" --lib --color always
+  clippy:
+      runs-on: ubuntu-22.04
+      steps:
+        - uses: actions/checkout@v3
+        - uses: actions-rs/cargo@v1
+          with:
+            command: clippy
+            args: --target "x86_64-unknown-linux-gnu" --lib --color always -- --deny warnings --allow unknown-lints -W clippy::expect_used -W clippy::panic -W clippy::unwrap_used
+  clippy-win:
+      runs-on: windows-2022
+      steps:
+        - uses: actions/checkout@v3
+        - uses: actions-rs/cargo@v1
+          with:
+            command: clippy
+            args: --target "x86_64-pc-windows-gnu" --lib --color always -- --deny warnings --allow unknown-lints
```

Comment:
Would it make sense to sync up `clippy-win and `clippy`? e.g. adding `clippy::expect_used`, `clippy::panic` as well as `clippy::unwrap_used`?

Reply:
Right now this will not work: https://github.com/NordSecurity/libtelio/actions/runs/5057020412/jobs/9075250873?pr=12#step:3:559 - I will create a ticket for that (fixing errors and syncing `clippy` and `clippy-win`). I will also create a ticket to create `clippy-mac` since mac specific code probably has the same issues and should be verified.

Reply:
Sounds like a plan!